frontend/public/components/utils/dropdown.jsx

```diff
@@ -459,7 +479,7 @@ export class ContainerDropdown extends React.PureComponent {
 ContainerDropdown.propTypes = {
   containers: PropTypes.oneOfType([PropTypes.object, PropTypes.array]).isRequired,
   currentKey: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  initContainers: PropTypes.object,
+  initContainers: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
```

Comment:
I'd rather not support two different formats in this component. It should be one or the other.

Can we just pass an object from the env var editor for all containers?

Reply:
I think allowing for flexibility in the component is better than reformatting the data we want to pass in, don't you think?

Reply:
I have to disagree here. I think it's better to accept one format to keep the component simpler and just give the component the data in the format it needs.